ipa/srs_precomp_test.go

```diff
@@ -2,46 +2,84 @@ package ipa
 
 import (
 	"bytes"
+	"fmt"
 	"testing"
 
+	"github.com/crate-crypto/go-ipa/bandersnatch/fr"
 	"github.com/crate-crypto/go-ipa/banderwagon"
+	"github.com/crate-crypto/go-ipa/common"
 )
 
 func TestPrecompSerde(t *testing.T) {
-	points := GenerateRandomPoints(2)
-	pcl := banderwagon.NewPrecomputeLagrange(points)
-	var buf bytes.Buffer
-
-	err := pcl.SerializePrecomputedLagrange(&buf)
-	if err != nil {
-		t.Fatal(err)
-	}
-	reader := bytes.NewReader(buf.Bytes())
-	deser, err := banderwagon.DeserializePrecomputedLagrange(reader)
-	if err != nil {
-		t.Fatal(err)
+	testNumPoints := []int{
+		2, 3, 7, 20, 64, // Arbitrary sizes
+		common.POLY_DEGREE, // Ethereum Verkle Tries case
 	}
```

Comment:
Instead of only doing this test for `2` (L11) points, we do it for different setups, and in particular cover the case of 256 which is important for Verkle Tries.

The other (2, 3, 7, 20, 64) cases is just to be sure the table computations are done properly even if we have less than 5 points or similar. In any case, the test should pass for any point size.